app/controllers/application_controller/tree_support.rb

```diff
@@ -20,6 +20,11 @@ def squash_toggle
     end
   end
 
+  def find_record
+    identify_record(params[:id], controller_name.classify) if %w(container_image host).include? controller_name
+    identify_record(params[:id], VmOrTemplate)
```

Comment:
what is the point of the first `identify-record` call here?  Was the intent something like this:
```ruby
if %w(container_image host).include? controller_name
  identify_record(params[:id], controller_name.classify)
else
  identify_record(params[:id], VmOrTemplate)
end
```